fem/fespace_ext.hpp

```diff
@@ -20,24 +20,22 @@ namespace mfem
 {
 
 // ***************************************************************************
-// * kFiniteElementSpace
-//  **************************************************************************
-class kFiniteElementSpace
+// * FiniteElementSpaceExtension
+// ***************************************************************************
+class FiniteElementSpaceExtension
 {
 private:
    FiniteElementSpace *fes;
    int globalDofs, localDofs;
    kernels::Array<int> offsets;
-   kernels::Array<int> indices, *reorderIndices;
+   kernels::Array<int> indices;
    kernels::Array<int> map;
 public:
-   kFiniteElementSpace(FiniteElementSpace*);
-   ~kFiniteElementSpace();
-   void GlobalToLocal(const Vector&, Vector&) const;
-   void LocalToGlobal(const Vector&, Vector&) const;
-   FiniteElementSpace& GetFes() { return *fes; }
+   FiniteElementSpaceExtension(FiniteElementSpace*);
```

Comment:
I think using a reference here is safer, there is no reason to allow `FiniteElementSpace*` to be `null`.